### PR TITLE
3383c42: Implementation of GunParts Catalog section

### PR DESCRIPTION
**Pull request by Logovas1994**

### CODE REVIEW

File: front/src/components/catalog/Catalog.tsx [hunk 1]
```diff
@@ -1,26 +1,18 @@
...
-    <div className={classes.catalog}>
-      {nfts.map((item: NFTCard) => (
-        <Link href={FRONT_CURRENT_PATH + ":3000/nft/" + item.nftCardId}>
-          <div className={classes.card} key={item.nftCardId}>
-            <NFTMicroCard item={item} />
-          </div>
-        </Link>
-      ))}
+    <div className={className}>
+      <div className={classes.catalogOfFour}>{children}</div>
```
Comment (nastyakuzusha) [inline front/src/components/catalog/Catalog.tsx]: thats not gonna work after merge. you should use layout as a parameter here - please, take a look at big_daddy_branch as its usage

File: front/src/components/catalog/CatalogStyles.ts [hunk 2]
```diff
@@ -1,32 +1,52 @@
 import { createStyles } from "@mantine/core";
```
Comment (nastyakuzusha) [inline front/src/components/catalog/CatalogStyles.ts]: please, use CatalogStyles from big_daddy_branch

File: front/src/components/gun-part-card/GunPartCardStyles.ts [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+    display: "flex",
+    flexDirection: "row",
+    justifyContent: "flex-start",
+    fontSize: theme.fontSizes.md,
+    lineHeight: 2,
+  },
+
+  brandIcon: {
+    margin: "0 4px 0 0",
+    width: "36px",
```
Comment (nastyakuzusha) [inline front/src/components/gun-part-card/GunPartCardStyles.ts]: please, use theme.size here

File: front/src/components/gun-part-card/GunPartCardStyles.ts [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+  brand: {
+    display: "flex",
+    flexDirection: "row",
+    justifyContent: "flex-start",
+    fontSize: theme.fontSizes.md,
+    lineHeight: 2,
+  },
+
+  brandIcon: {
+    margin: "0 4px 0 0",
```
Comment (nastyakuzusha) [inline front/src/components/gun-part-card/GunPartCardStyles.ts]: pls use theme.spacing here

File: front/src/components/gun-part-card/GunPartCardStyles.ts [hunk 5]
```diff
@@ -0,0 +1,39 @@
+import { createStyles } from "@mantine/core";
+
+export const useStyles = createStyles((theme) => ({
+  card: {
+    margin: theme.spacing.xs,
+    maxWidth: "242px",
+    minHeight: "370px",
+    boxShadow: "0px 4px 40px rgba(0, 0, 0, 0.1)",
```
Comment (nastyakuzusha) [inline front/src/components/gun-part-card/GunPartCardStyles.ts]: please define shadow in theme and use it

File: front/src/components/gun-part-card/GunPartCardStyles.ts [hunk 6]
```diff
@@ -0,0 +1,39 @@
+import { createStyles } from "@mantine/core";
+
+export const useStyles = createStyles((theme) => ({
+  card: {
+    margin: theme.spacing.xs,
+    maxWidth: "242px",
```
Comment (nastyakuzusha) [inline front/src/components/gun-part-card/GunPartCardStyles.ts]: please, use theme here

File: front/src/components/gun-part-card/GunPartCardStyles.ts [hunk 7]
```diff
@@ -0,0 +1,39 @@
...
+    minHeight: "370px",
```
Comment (nastyakuzusha) [inline front/src/components/gun-part-card/GunPartCardStyles.ts]: this is not gonna work, take a look at catalog styles in big_daddy_branch

File: front/src/pages/NFTCatalogStyles.tsx [hunk 8]
```diff
@@ -1,9 +1,16 @@
 import { createStyles } from "@mantine/core";
 
 export const useStyles = createStyles((theme) => ({
-    page: {
-        display: "flex",
-        alignItems: "flex-start",
-        justifyContent: "center",
+  page: {
```
Comment (nastyakuzusha) [inline front/src/pages/NFTCatalogStyles.tsx]: please, dont change NFTCatalogStyles in this PR

**Comment by nastyakuzusha** [comment]
Screenshot 2022-11-25 at 12 57 05
1. Where is the header? 
2. Whats wrong with cards width?
3. Cards content should be aligned to the bottom
4. Design doesnt contain card borders

File: front/styles/globals.css [hunk 9]
```diff
@@ -14,12 +14,12 @@ a {
   box-sizing: border-box;
 }
 
-@media (prefers-color-scheme: dark) {
+/*@media (prefers-color-scheme: dark) {
```
Comment (AbrosimovVladislav) [inline front/styles/globals.css]: wtf?

Comment (Logovas1994) [reply front/styles/globals.css]: I turned off the theme change to dark and light, now we have only light.
If you are against it, I will uncomment.

**Pull request merged**